### DISCUSSION TITLE
Using Link or router to go to homepage goes to dynamic page instead

### DISCUSSION THREAD

**Discussion by elliot619** [vercel/next.js#41146] [Help]
Hello everybody,
I'm having a weird behavior when using either Link or router from NextJS to navigate to homepage when using a non-default locale (i.e.: en-CA or fr-CA), because instead of going to home it goes to my dynamic page [...slug].tsx.

Using a regular anchor or directly typing the url on the browser works fine.
Is there any way to override this behavior and actually go to home page using Link or router?

Here is how I'm using it:

![image](https://user-images.githubusercontent.com/8999155/193707532-7f0d8283-8381-43e1-beb4-7740bb4f124c.png)

And here is the error I get when clicking on the rendered link from above's code:
![image](https://user-images.githubusercontent.com/8999155/193707583-7bcfca0d-977e-43d1-a276-a4e1c57c3397.png)

Thanks a lot

**Comment by icyJoseph** [comment]
Hi,

Do you think you could make a [stackblitz](https://stackblitz.com/) reproduction?

I've tried with `nl-NL` and `fr` locales, having a `[...slug]` at the root, with and without `middleware`, and it works fine to navigate back home from any other page.

Strange that the link complains about `as` since the code snippet you show is not using it. Share some more code!
